maps/biter_battles_v2/item_costs.lua

```diff
@@ -33,6 +33,42 @@ ItemCosts.raw_costs = {
     -- So that is 150 oil/sec == 30 ore/sec.
     ['crude-oil'] = { cost = 2, crafting_time = 0.1 / 0.8, icon = '[fluid=crude-oil]' },
     ['water'] = { cost = 0, crafting_time = 1 / 1200, icon = '[fluid=water]' },
+
+    --space
+    ['ice'] = { cost = 1, crafting_time = 2, icon = '[item=ice]' },
+
+    --gleba
+    ['jelly'] = { cost = 7, crafting_time = 1, icon = '[item=jelly]' },
+    ['yumako-mash'] = { cost = 14, crafting_time = 1, icon = '[item=yumako-mash]' },
+
+    --vulcanus
+    ['tungsten-ore'] = { cost = 20, crafting_time = 5, icon = '[item=tungsten-ore]' },
+    ['calcite'] = { cost = 1, crafting_time = 2, icon = '[item=calcite]' },
+
+    --fulgora
+    ['holmium-ore'] = { cost = 15, crafting_time = 1 / 1200, icon = '[item=holmium-ore]' },
```

Comment:
holmium ore has 1% yield from scrap so takes a lot of crafting to make.

but if you're doing scrap already, not just to get the holmium, then the crafting is free.

Reply:
myeah.. i didnt exactly know what those values were used for so i figured i'd just make something up and if anyone notices and thinks its weird.. they can just be changed, i mean look at the crafting times what even does that do? i saw no discernable difference between 200 or 1/1200

Reply:
Yeah I'm not sure what it does either. Fair enough.